INSTRUCTIONS.md

```diff
@@ -2,66 +2,83 @@
 
 This document provides instructions and explanations for completing the BCI workshop given by BCI Montréal in May 2015. Specifically, it will guide you through the installation of the necessary software, the configuration of the device we will use, and the two exercises that are at the core of this workshop.
 
-This workshop is intended for people with no or limited experience with Brain-Computer Interfaces (BCIs). The workshop will teach them the basic principles that are necessary to "hack" and develop new applications with BCIs: background on brain activity and brain activity measurement with EEG, structure of a BCI, feature extraction and machine learning. Two hands-on exercises will allow the participants to 1) visualize their EEG signals and some relevant features, and 2) experiment with a very simple BCI design. This should give the participants sufficient knowledge to understand the advantages and limitations of current BCIs, and to devise their own applications. 
+This workshop is intended for people with no or limited experience with Brain-Computer Interfaces (BCIs). The workshop will teach them the basic principles that are necessary to "hack" and develop new applications with BCIs: background on brain activity and brain activity measurement with EEG, structure of a BCI, feature extraction and machine learning. Two hands-on exercises will allow the participants to 1) visualize their EEG signals and some relevant features, and 2) experiment with a very simple BCI design. This should give the participants sufficient knowledge to understand the advantages and limitations of current BCIs, and to devise their own applications.
 
-## Installation of software for the workshop
+## Programming languages for the workshop exercises
+The material for this workshop is provided in two flavors: **Python scripts** and **Matlab scripts**.  
+* **Python***: a popular, multi-purpose powerful, free, open and simple to read scripting language.
+* **Matlab**: very popular in academia, technical programing oriented, license required, not-open.
 
-The workshop is based on the following tools:
+Use the one that is more convenient for you.
 
-1. **Python 2 or Python 3, (and packages)**: a popular, powerful, free and simple to read scripting language
-2. **Muse SDK**: the software development kit provided to play around with the Muse EEG headband
-3. **MuLES**: an EEG server that allows device-agnostic applications
+***** In this workshop, the Python scripts are compatible with Python 2 and Python 3.
```

Comment:
That was the original idea, a bold star in both cases, in the Atom Markdown implementation the 5 start are taken as a bold star, but not in GitHub;  changing to the underscore syntax for bold `__*__`